api/model.py

```diff
@@ -50,9 +51,13 @@ def post(self):
         result = {'status': 'error'}
 
         args = image_parser.parse_args()
+
+        if not args['image'].mimetype.endswith(('jpg', 'jpeg', 'png')):
+            abort(400, 'Invalid file type/extension')
```

Comment:
Perhaps add that only JPEG and PNG are supported in the error message? Such as "Invalid file type/extension. Please provide a JPEG or PNG image" or similar.